src/interfaces/Config/ITagConfig.ts

```diff
@@ -0,0 +1,16 @@
+interface ITagConfig {
+  responsive?: boolean;
+  responsiveClass?: string;
+  responsiveWidth?: boolean;
+  responsivePlaceholder?: string;
```

Comment:
We decided to have IResponsiveConfig as a child of ITagConfig

Reply:
I wasn't aware of that